tests/roxctl/bats-tests/local/deployment-bundles-psps.bats

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bats
+
+load "../helpers.bash"
+
+out_dir=""
+
+setup_file() {
+    # remove binaries from the previous runs
+    [[ -n "$NO_BATS_ROXCTL_REBUILD" ]] || rm -f "${tmp_roxctl}"/roxctl*
+    echo "Testing roxctl version: '$(roxctl-release version)'" >&3
+
+    command -v grep || skip "Command 'grep' required."
+    [[ -n "${API_ENDPOINT}" ]] || fail "Environment variable 'API_ENDPOINT' required"
+    [[ -n "${ROX_PASSWORD}" ]] || fail "Environment variable 'ROX_PASSWORD' required"
+}
+
+setup() {
+    out_dir="$(mktemp -d -u)"
+}
+
+teardown() {
+    rm -rf "$out_dir"
+}
+
+central_bundle_psp_enabled() {
+    local cluster_type="$1"
+    shift
+    local bundle_dir="${out_dir}/bundle-central-$RANDOM"
+    roxctl-release -e "$API_ENDPOINT" -p "$ROX_PASSWORD" central generate "${cluster_type}" pvc "$@" --output-dir="${bundle_dir}"
```

Comment:
[nit] No need to use api endpoint and password with `central generate` because it works locally.

```suggestion
    roxctl-release central generate "${cluster_type}" pvc "$@" --output-dir="${bundle_dir}"
```